packages/ssz/test/perf/misc/switch_if_cost.test.ts

```diff
@@ -5,16 +5,16 @@ describe("big if vs obj", () => {
 
   function fnIf(n: number): number {
     if (n === 0) return 0;
-    else if (n === 1) return 1;
-    else if (n === 2) return 2;
-    else if (n === 3) return 3;
-    else if (n === 4) return 4;
-    else if (n === 5) return 5;
-    else if (n === 6) return 6;
-    else if (n === 7) return 7;
-    else if (n === 8) return 8;
-    else if (n === 9) return 9;
-    else throw Error("bad n");
+    if (n === 1) return 1;
+    if (n === 2) return 2;
+    if (n === 3) return 3;
+    if (n === 4) return 4;
+    if (n === 5) return 5;
+    if (n === 6) return 6;
+    if (n === 7) return 7;
+    if (n === 8) return 8;
+    if (n === 9) return 9;
+    throw Error("bad n");
```

Comment:
This is a perfect example of where no bracket conditionals are ok and even desirable.  But this type of code segment is about the only exception imho